src/cuckoocache.h

```diff
@@ -458,10 +458,10 @@ class cache
      * contains returns a bool set true if the element was found.
      *
      * @param e the element to check
-     * @param erase
+     * @param erase whether to consider setting the garbage collect flag
```

Comment:
not sure if consider is the right verb. It implies theres some other check.

Reply:
Replaced `consider` with `attempt`

Reply:
maybe

>      * @param erase whether to attempt setting the garbage collect flag (element may not exist or flag may already be set) 

to make it clear which cases the 'attempt' may fail.